apps/main/src/components/PageCreatePool/PoolType/index.tsx

```diff
@@ -29,9 +29,9 @@ const PoolType = ({ chainId }: Props) => {
               descriptionName={t`Stableswap`}
               description={t`Bonding Curve specialising in pegged assets.`}
               handleClick={() => updateSwapType(STABLESWAP, chainId)}
-              disabled={!networks[chainId].stableSwapFactory && !networks[chainId].stableSwapNg}
+              disabled={!networks[chainId].stableswapFactory && !networks[chainId].stableswapFactoryOld}
```

Comment:
it might be more readable and a little bit more performance to extract a variable `const network = neworks[chainId]`